.github/workflows/release.yml

```diff
@@ -0,0 +1,315 @@
+name: Build + draft release
+
+# CR erodkin: delete this!
+on: pull_request
+
+# (erodkin) this is needed to create the release, per https://github.com/softprops/action-gh-release/issues/236
+permissions:
+  contents: write
+
+jobs:
+  prepare:
+    if: github.repository_owner == 'viamrobotics'
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.which_version.outputs.version }}
+      sha: ${{ steps.commit.outputs.commit_long_sha }}
+    steps:
+      - name: Check if organization member
+        id: is_organization_member
+        uses: jamessingleton/is-organization-member@1.0.1
+        with:
+          organization: viamrobotics
+          username: ${{ github.actor }}
+          token:  ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Cancelling - user not part of organization
+        uses: andymckay/cancel-action@0.2
+        if: |
+          steps.is_organization_member.outputs.result == 'false'
+
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Which Version
+        id: which_version
+        shell: bash
+        run: echo "version=$(grep 'set(CMAKE_PROJECT_VERSION' CMakeLists.txt | sed 's/)//' | awk '{print $2}')" >> $GITHUB_OUTPUT
```

Comment:
Did you investigate my suggestion to derive the `CMAKE_PROJECT_VERSION` from tag metadata via `git describe`?

Reply:
I did a bit but I ended up getting way too deep without feeling I was super confident in what I was doing. I'm sure that a `git describe` approach is possible but this one felt easier such that we could get _something_ out the door fast, and also I don't believe this does anything to preclude use of `git describe` in the future. If you have strong feelings that we should be using the `git describe` approach from jump then I'm certainly willing to reconsider!

Reply:
We could certainly tag this commit, I was planning on making a versioned release a la the Python SDK and rust-utils. I think the plan would be to do biweekly "releases" in the same way that would bundle up the source code, so I expect we'd be leaving `0.0.1` behind pretty soon (though not hitting `0.1.0` for some time).

Reply:
I'd like to suggest that we call the tag either `releases/0.0.1` or `releases/v0.0.1`, in the interest of making it easier to ensure we pick up a `release` tag with a future `git describe`-based workflow, by using the `--match` flag.